atlp-rwanda/atlp-pulse-fn
# Fix Coverage

# PR Description
This PR fixes the coverage by increasing it above 80% 

# Description of tasks that were expected to be completed

1. Checking if there are the task which needs to be tested and write the tests
2. Checking open branches if they meet with the coverage needed(no reduction of coverage) if so , write test to increase the coverage
3. Cant' be merged if coverage is below 80%

# How has this been tested?

1. Clone this RP
2. Install needed Depencencies by runing `npm install`
4. Add environment variables check `.env.example` if needed
5. After run `npm run coverage` 

# Number of Commits
1
# Screenshots (If appropriate)
N/A

# Please check this Checklist before you submit your PR:
